File: src/main/java/io/github/oleksivio/telegram/bot/core/configuration/LibraryConfiguration.java
package io.github.oleksivio.telegram.bot.core.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@Configuration
public class LibraryConfiguration {

    @Bean
    public RestTemplate restTemplate() {
        return new RestTemplate();
    }

}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/core/configuration/LibraryConfiguration.kt
package io.github.oleksivio.telegram.bot.core.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.client.RestTemplate

@Configuration
class LibraryConfiguration {

    @Bean
    fun restTemplate(): RestTemplate {
        return RestTemplate()
    }

}
